app/lambda/list_sessions/lambda_function.py

```diff
@@ -7,6 +7,27 @@
 from models import Session
 from constant_variables import *
 
+class DecimalEncoder(json.JSONEncoder):
```

Comment:
Do we need this class? If we do, should this be a part of a separate file called (eg utility.py)

Reply:
We do. Done. https://github.com/aws-samples/medical-transcription-analysis/blob/f6051ba5393880e46671671fc967c4d679707992/app/lambda/list_sessions/lambda_function.py#L8-L12